### DISCUSSION TITLE
Lista iptv

### DISCUSSION THREAD

**Discussion by MaxTheWinner** [iptv-org/community#1022] [Show and tell]
A lista que você forneceu nao esta funcionando!

**Comment by BellezaEmporium** [comment]
If you're still using the NSFW playlist, please note it has been removed since recently. Please use the other link.

**Comment by Artiom97es** [comment]
any particular reason?

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#how-to-report-a-broken-stream
